app/src/components/collectionLanes/collectionLanes.tsx

```diff
@@ -6,38 +6,67 @@ import {
   Heading,
   Spacer,
   Link,
+  Icon,
 } from "@nypl/design-system-react-components";
 import styles from "./Collectionlanes.module.css";
 import useBreakpoints from "../../hooks/useBreakpoints";
 import CollectionCard from "../cards/collectionCard";
 import CollectionDataType from "../../types/CollectionDataType";
 import { CollectionCardModel } from "../../models/collectionCard";
 import DCSimpleGrid from "../dcSimpleGrid/dcSimpleGrid";
+import { headerBreakpoints } from "@/src/utils/breakpoints";
 
 const CollectionLanes = ({ lanesWithNumItems, seeMoreLink }) => {
   const { isLargerThanLargeTablet } = useBreakpoints();
   const lanes = lanesWithNumItems?.map((lane, key) => (
     <Box className={styles.lane} data-testid={lane.slug} mt="xxl" key={key}>
       <Flex alignItems="baseline">
-        <Heading id={`row-heading-${lane.slug}`} level="h2" size="heading3">
+        <Heading
+          id={`row-heading-${lane.slug}`}
+          sx={{
+            [`@media screen and (min-width: ${headerBreakpoints.smTablet}px)`]:
+              {
+                width: "90%",
+              },
+          }}
+          level="h2"
+          size="heading3"
+        >
           {lane.name}
         </Heading>
         <Spacer />
         <Link
           id={`row-see-more-${lane.slug}`}
-          type="standalone"
           href={`${seeMoreLink}/${lane.slug}`}
           aria-label={`See more ${lane.name}`}
           hasVisitedState
+          isUnderlined={false}
           __css={{
-            display: { sm: "none", md: "inline" },
-            color: "ui.primary.link",
-            fontWeight: "500 !important",
-            alignItems: "center",
-            _hover: { textDecoration: "underline 1px dotted !important" },
+            paddingLeft: "l",
+            marginBottom: "s",
+            whiteSpace: "nowrap",
+            justifyContent: "flex-end",
+            display: "none",
+            " > svg": {
+              fill: "ui.link.primary",
+            },
+            [`@media screen and (min-width: ${headerBreakpoints.smTablet}px)`]:
+              {
+                display: "inline",
+              },
+            color: "ui.link.primary",
+            fontWeight: "medium",
+            alignItems: "end",
```

Comment:
`align-self` is the right rule. This one works if the container has `display: "flex"` for its children, so you can remove this.

Reply:
I see the other "see more" link has "flex" so maybe this carried over from that one?